docs/commands.md

```diff
@@ -53,8 +53,8 @@ This input should be used sparingly and with caution, as it involves interacting
 
 #### Subdomain Testing and `subdomain-url`
 
-The `subdomain-url` parameter is utilized when testing subdomain support in your IPFS gateway. It can be set to any domain that your gateway permits.
-During testing, the suite keeps connecting to the `gateway-url` while employing HTTP techniques to simulate requests as if they were sent to the subdomain.
+The `subdomain-url` parameter is utilized when testing subdomain support in your IPFS gateway. It can be set to any domain that your gateway dedicated and safelisted for [Subdomain gateway](https://specs.ipfs.tech/http-gateways/subdomain-gateway/) feature.
```

Comment:
```suggestion
The `subdomain-url` parameter is utilized when testing subdomain support in your IPFS gateway. It can be set to any domain that your gateway has dedicated to and safelisted for the [Subdomain gateway](https://specs.ipfs.tech/http-gateways/subdomain-gateway/) feature.
```